i18n/js_to_json.js

```diff
@@ -12,6 +13,7 @@ const PATH_OUTPUT = path.resolve(__dirname, '../msg/json/en.json');
 // Match function
 const match = function (str) {
     if (str.indexOf('Blockly.Msg.') !== 0) return false;
+    assert.notStrictEqual(str.indexOf("';"), str.length - 2, `[${str}] uses single quoted string, should use double quotes.`);
     if (str.indexOf('";') !== str.length - 2) return false;
```

Comment:
Should there be a similar assertion for this line? It looks like this will fail silently if " is not the second to last character in the line.

Reply:
Also, a general question, perhaps not directly related to this PR: why not use a regex for this 'match' function?

E.g. something like `/^Blockly\.Msg\.*";$/`

Reply:
I think it's for performance. regex is slower and the message strings do have a very predictable format.